0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<p>This also mean that it's ok to use global <strong>constants </strong>inside jax functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>

Reply:
<p>Nice point. Will add a sentence on this. </p>